github/Requester.pyi

```diff
@@ -47,14 +48,19 @@ class HTTPSRequestsConnectionClass:
     ) -> None: ...
 
 class Requester:
+    __installation_authorization: Optional[InstallationAuthorization]
+    __app_id: Optional[Union[int, str]]
+    __app_private_key: Optional[str]
+    __app_installation_id: Optional[int]
+    __app_token_permissions: Optional[dict]
     def DEBUG_ON_RESPONSE(
         self, statusCode: int, responseHeader: Dict[str, str], data: str
     ) -> None: ...
     def NEW_DEBUG_FRAME(self, requestHeader: Dict[str, str]) -> None: ...
     def __check(
         self,
         status: int,
-        responseHeader: Dict[str, Any],
+        responseHeaders: Dict[str, Any],
```

Comment:
There are still unrelated changes in this file, but I understand it's hard to untangle them.